src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -458,8 +458,8 @@ private static LogEntry WriteEntry(EventLogLevel eventLogLevel, Exception innerE
             // Prefix header (library version, DateTime, whether PII is ON/OFF) to the first message logged by Wilson.
             if (!_isHeaderWritten)
             {
-                string headerMessage = string.Format(CultureInfo.InvariantCulture, "Microsoft.IdentityModel Version: {0}. Date {1}. {2}",
-                    typeof(IdentityModelEventSource).Assembly.GetName().Version.ToString(),
+                string headerMessage = string.Format(CultureInfo.InvariantCulture!, "Microsoft.IdentityModel Version: {0}. Date {1}. {2}",
+                    typeof(IdentityModelEventSource).Assembly!.GetName().Version!.ToString(),
```

Comment:
why are these needed?

Reply:
IDE throws error code CS8602-Dereference of a possibly null reference. Version is not nullable in System.Reflection so it causes noise